sdk/cosmos/src/clients/cosmos_client.rs

```diff
@@ -20,18 +22,11 @@ const AZURE_VERSION: &str = "2018-12-31";
 const VERSION: &str = "1.0";
 const TIME_FORMAT: &str = "%a, %d %h %Y %T GMT";
 
-use azure_core::*;
-
-#[derive(Clone, Debug)]
-enum TransportStack {
```

Comment:
I don't understand why you would have an `HttpClient` or a `Pipeline`. May be it can this can be reintroduced in a subsequent pull request.